src/transformers/__init__.py

```diff
@@ -778,7 +778,7 @@
     _import_structure["image_utils"] = ["ImageFeatureExtractionMixin"]
     _import_structure["models.beit"].extend(["BeitFeatureExtractor", "BeitImageProcessor"])
     _import_structure["models.bit"].extend(["BitImageProcessor"])
-    _import_structure["models.blip"].extend(["BlipImageProcessor"])
+    _import_structure["models.blip"].extend(["BlipFeatureExtractor", "BlipImageProcessor"])
```

Comment:
No need to add the feature extractor, you can use the image processor for the pipeline.

Reply:
Yes, but the pipeline still builds a feature_extractor(which is actually a blipimageprocessor)?

Reply:
Now not anymore ;) please rebase on main